src/Uno.UI.Tests/App/Xaml/Subclassed_Dictionary.xaml.cs

```diff
@@ -7,6 +7,7 @@
 using Uno.UI.Tests.ViewLibrary;
 using Windows.Foundation;
 using Windows.Foundation.Collections;
+using Windows.Networking.Sockets;
```

Comment:
slipped in?

Reply:
For some reason, intellisense adds those without apparent reason... will remove.